core-parser/src/main/java/org/uniprot/core/parser/tsv/uniparc/UniParcEntryCrossRefValueMapper.java

```diff
@@ -0,0 +1,74 @@
+package org.uniprot.core.parser.tsv.uniparc;
+
+import java.time.LocalDate;
+import java.time.temporal.ChronoUnit;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import org.uniprot.core.parser.tsv.EntityValueMapper;
+import org.uniprot.core.uniparc.UniParcCrossReference;
+import org.uniprot.core.util.Utils;
+
+/**
+ * @author sahmad
+ * @created 24/03/2021
+ */
+public class UniParcEntryCrossRefValueMapper implements EntityValueMapper<UniParcCrossReference> {
+    @Override
+    public Map<String, String> mapEntity(UniParcCrossReference entity, List<String> fieldNames) {
+        Map<String, String> fieldValue = new HashMap<>();
+        fieldValue.put("database", entity.getDatabase().getName());
+        fieldValue.put("accession", entity.getId());
+        fieldValue.put("gene", getNullSafeValue(entity.getGeneName()));
+        fieldValue.put("ncbiGi", getNullSafeValue(entity.getNcbiGi()));
+        fieldValue.put(
```

Comment:
Suggestion: move these if to a method.
Fields:  organism, organism_id and version